rflx/pyrflx/typevalue.py

```diff
@@ -555,6 +555,7 @@ def __init__(
             self._last_field = INITIAL.name
         else:
             self._preset_fields(INITIAL.name)
+        self.message_last_name = state.last_name if state else Last("Message")
```

Comment:
No, but this is intended. It prevents instantiating `Last("Message")` multiple times by reusing this object.

Reply:
What means "this is intended"? Will this be changed in this PR? In the current implementation you could just use a constant for `Last("Message")` instead of storing `last_name` in `Field`.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/424/commits/1de5618480146e97d5496c558964e5df6de053f6.